src/Kobo/SystemDialog.cpp

```diff
@@ -63,6 +71,13 @@ SystemWidget::Prepare(ContainerWindow &parent, const PixelRect &rc) noexcept
 #endif
   AddButton("Export USB storage", [this](){ ExportUSBStorage(); });
   SetRowEnabled(USB_STORAGE, !IsKoboOTGKernel());
+
+  if(KoboCanChangeBacklightBrightness()) {
+    increase_backlight_brightness = AddButton("Increase Backlight Brightness", [this]() { IncreaseBacklightBrightness(); });
+    decrease_backlight_brightness = AddButton("Decrease Backlight Brightness", [this]() { DecreaseBacklightBrightness(); });
```

Comment:
These lines will be executed conditionally, but if `!KoboCanChangeBacklightBrightness()`, then those control indexes don't exit. That's not a problem now, but for the next guy who adds new controls to this form.
To avoid this, add `RowFormWidget::AddDummy()` calls to the `else` branch. This reserves index numbers, but doesn't actually add controls.